idaes/models_extra/column_models/reboiler.py

```diff
@@ -269,9 +269,31 @@ def rule_boilup_ratio(self, t):
                 self.flowsheet().time, rule=rule_boilup_ratio
             )
 
-        self._make_ports()
+        self.add_inlet_port()
 
-        self._make_splits_reboiler()
+        # Outlet ports that always exist irrespective of reboiler type
+        self.bottoms = Port(noruleinit=True, doc="Bottoms stream.")
+
+        self.vapor_reboil = Port(
+            noruleinit=True,
+            doc="Vapor outlet stream that is returned to " "to the bottom tray.",
+        )
+
+        make_phase_split(
+            self.control_volume,
+            port=self.bottoms,
+            phase=self._liquid_set,
+            side_sf=1,
```

Comment:
Why is 1 hard coded here? A comment here will be helpful.

Reply:
Added a comment